packages/tangle/tangle.go

```diff
@@ -52,7 +49,6 @@ type Tangle struct {
 	Events                *Events
 
 	setupParserOnce sync.Once
-	fifoScheduling  typeutils.AtomicBool
 	shutdownSignal  chan struct{}
```

Comment:
Is `shutdownSignal` still used? It was introduced to cancel the `sleep` in the scheduler switching, which has been removed as part of this PR.
Of course, it also does not hurt either to have this in for future expansions.